egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -150,13 +150,15 @@ def AddConvolutionLayer(config_lines, name, input,
     components = config_lines['components']
     component_nodes = config_lines['component-nodes']
 
-    conv_init_string = "component name={0}_conv type=ConvolutionComponent input-x-dim={1} input-y-dim={2} input-z-dim={3} filt-x-dim={4} filt-y-dim={5} filt-x-step={6} filt-y-step={7} input-vectorization-order={8}".format(name, input_x_dim, input_y_dim, input_z_dim, filt_x_dim, filt_y_dim, filt_x_step, filt_y_step, input_vectorization)
+    conv_init_string = ("component name={0}_conv type=ConvolutionComponent "
+                       "input-x-dim={1} input-y-dim={2} input-z-dim={3} "
```

Comment:
Something like (this)[https://github.com/kaldi-asr/kaldi/blob/master/egs/wsj/s5/steps/nnet3/nnet3_train_lib.py#L226].